docs/src/main/asciidoc/spring-cloud-bus-pubsub.adoc

```diff
@@ -0,0 +1,36 @@
+== Spring Cloud Bus
+
+Spring Cloud Bus integration with GCP Pub/Sub is as simple as importing two Spring Cloud dependencies: `spring-cloud-bus` and `spring-cloud-gcp-pubsub-stream-binder`.
```

Comment:
Can we just create a simple `spring-cloud-gcp-starter-bus`?

Reply:
Yep